python_modules/libraries/dagster-dbt/dagster_dbt_tests/dbt_projects/test_dagster_metadata/packages.yml

```diff
@@ -1,4 +1,6 @@
 # We keep use `packages.yml` for compatability with `dbt-core==1.5.*`.
 # Once we remove support for that version, we should rename this file to `dependencies.yml`
 packages:
-  - local: "../../../dbt_packages/dagster"
+  - git: "https://github.com/dagster-io/dagster.git"
```

Comment:
Cut over to published version so we can remove this package now (or in the future) and the warning test will still work.